project2/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="card">
+        <div class="content">
+            <p class="moviename">Her</p>
```

Comment:
This feels like it should be more significant than a `p` element, consider making it a heading?
```suggestion
            <h2 class="moviename">Her</h2>
```